ops/framework.py

```diff
@@ -1191,8 +1189,9 @@ def __get__(self,
                 if bound is not None:
                     # the StoredState instance is being stored in two different
                     # attributes -> unclear what is expected of us -> bail out
-                    raise RuntimeError("StoredState shared by {0}.{1} and {0}.{2}".format(
-                        cls.__name__, self.attr_name, attr_name))
+                    raise RuntimeError(
+                        f"StoredState shared by {cls.__name__}.{self.attr_name} and "
+                        f"{cls.__name__}.{attr_name}")
```

Comment:
Maybe reduce the number of changes in the future by making this single quotes now?

```suggestion
                    raise RuntimeError(
                        f'StoredState shared by {cls.__name__}.{self.attr_name} and '
                        f'{cls.__name__}.{attr_name}')
```